app/controllers/ems_cloud_controller.rb

```diff
@@ -75,7 +75,8 @@ def update_ems_button_validate(verify_ems = nil)
     set_ems_record_vars(verify_ems, :validate)
     @in_a_form = true
 
-    result, details = verify_ems.authentication_check(params[:cred_type], :save => false)
+    result, details = verify_ems.authentication_check(params[:cred_type],
+                                                      :save => params["#{params[:cred_type]}_password".to_sym].nil?)
```

Comment:
maybe the condition that was there before was is needed? save = params[:id] == "new" ? false : true

otherwise it could save for non created ems?